### DISCUSSION TITLE
psycopg2 Installation failed

### DISCUSSION THREAD

**Discussion by jlij** [beeware/briefcase#1261] [Q&A] [answered]
[briefcase.2023_05_05-09_28_54.build.log](https://github.com/beeware/briefcase/files/11402716/briefcase.2023_05_05-09_28_54.build.log)

**Comment by freakboy3742** [comment, marked as answer]
Firstly - posting a log message isn't a question. It might be useful supporting evidence, but if you want an answer, you need to ask a question.

Secondly, the [most recent discussion in this forum that wasn't posted by yourself](https://github.com/beeware/briefcase/discussions/1259) is literally the same topic as this one, and contains a complete answer.
